libraries/redcore/layouts/webservice/documentationoperation/example.php

```diff
@@ -221,45 +205,16 @@
 				endif; ?>
 			</small>
 		</div>
-	</div>
-
-	<?php if ($soapEnabled): ?>
-		<strong><?php echo JText::_('LIB_REDCORE_API_HAL_WEBSERVICE_DOCUMENTATION_SOAP'); ?></strong><br />
-		<h5>
-			<span class="label label-success">
-				$client-><?php echo $soapFunction; ?>
-			</span>
-			<br /><br />
-			<?php echo $url . '&api=soap'; ?>
-		</h5>
-		<em><?php if ($operationName == 'read item'):
-			$arrayIds = array();
-
-			foreach ($ids as $id):
-				$arrayIds[] = "'" . $id . "' => '1'";
-			endforeach;
-
-			echo JText::sprintf('LIB_REDCORE_API_HAL_WEBSERVICE_DOCUMENTATION_SOAP' . $noteName . '_NOTE', implode(', ', $ids),
```

Comment:
can you please remove those key string as well since they are not being used anymore...